src/core/mount.c

```diff
@@ -1750,9 +1783,54 @@ static int mount_load_proc_self_mountinfo(Manager *m, bool set_flags) {
                 if (!device || !path)
                         continue;
 
+                if (ignore_mount(path, fstype))
+                        continue;
+
```

Comment:
I feel the helper definition and the move from mount_setup_unit() to mount_load_proc_self_mountinfo() of the ignore mount check is a separate task and doing this in a separate commit would make the later commit easier to read as well as making the ignore change itself clear on what is being done.

I think that would also eliminate the need for the #if 0 ... #endif altogether since that standalone commit would speak for itself quite clearly about what's being done.

Reply:
Yes that `#if 0` should probably go away.